module1/src/components/Counter.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, ComponentClass } from "react";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: this.props.initValue || 0,
```

Comment:
this.props.initValue - what is this? Where did you use it?

Reply:
<Counter initValue={10}></Counter>